modules/serverless-functions-adding-annotations.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-annotations.adoc
+
+[id="serverless-functions-adding-annotations_{context}"]
+= Adding annotations to a function
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For every annotation that you want to add, add the following YAML to the `annotations` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+annotations:
+  color: "red"
```

Comment:
```suggestion
  author: "alice@example.com"
```

Reply:
Went with Ashleigh's suggestion, it the resultant step is more concise.